vendor/github.com/docker/swarmkit/vendor.conf

```diff
@@ -45,7 +45,7 @@ github.com/sirupsen/logrus v1.0.6
 github.com/beorn7/perks 3a771d992973f24aa725d07868b467d1ddfceaf
 github.com/cloudflare/cfssl 1.3.2
 github.com/dustin/go-humanize 8929fe90cee4b2cb9deb468b51fb34eba64d1bf0
-github.com/fernet/fernet-go 1b2437bc582b3cfbb341ee5a29f8ef5b42912ff2
+github.com/fernet/fernet-go 9eac43b88a5efb8651d24de9b68e87567e029736
```

Comment:
We probably want this one bumped as well (or merge https://github.com/moby/moby/pull/39269 first)